src/countervalues/logic.test.js

```diff
@@ -28,7 +40,39 @@ describe("API sanity", () => {
       });
       expect(value).toBeDefined();
     }
+    expect(
+      calculate(state, { from: bitcoin, to: usd, value: 10000000 })
+    ).toBeGreaterThan(1000);
   });
+  test("resilience when no network with a cache", async () => {
+    const userSettings = {
+      trackingPairs: [
+        {
+          from: bitcoin,
+          to: usd,
+          startDate: new Date(now - 200 * 24 * 60 * 60 * 1000),
```

Comment:
Is this arbitrary?

Reply:
yeah. like many tests. it's going to ask 200 days of data like if account was 200 days old.